manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -5186,121 +5186,467 @@
+                      </varlistentry>
+
+                      <varlistentry>
+                        <term><guilabel>Mehrere Kursquellen</guilabel></term>
+
+                        <listitem>
+                          <para>Kursquellen wie <quote>Europa</quote> sollten verwendet werden, wenn eine Ausfallsicherung
+                            gegenüber mehreren Datenquellen erwünscht ist.
+                          </para>
+
+                          <caution>
+                            <para>Die Option <guilabel>Mehrere Kursquellen</guilabel> ist nur sinnvoll, wenn alle ausgewählten
+                              Quellen das gleiche Symbol für Ihre Wertpapier verwenden.
+                            </para>
+                          </caution>
+                        </listitem>
+                      </varlistentry>
+
+                      <varlistentry>
+                        <term><guilabel>unbekannt</guilabel></term>
+<!-- Maintainers: To add, modify, remove sources
+edit https://github.com/Gnucash/gnucash/blob/maint/libgnucash/engine/gnc-commodity.c
+and update the appendix. -->
+                        <listitem>
+                          <para>&app; hat hier kein Hintergrundwissen über die Quelle. Das kann zwei verschiedene Gründe haben.
+                            Sie versuchen eine Quelle zu verwenden, die
+                            <itemizedlist>
+                              <listitem>
+                                <para>es <emphasis>nicht mehr</emphasis> in &app-fq; gibt oder
+                                </para>
+                              </listitem>
+
+                              <listitem>
+                                <para>neu in &app-fq; ist, aber <emphasis>noch nicht</emphasis> in &app; eingepflegt wurde.
+                                </para>
+                              </listitem>
+                            </itemizedlist>
+                          </para>
+                        </listitem>
+                      </varlistentry>
+                    </variablelist>
+                  </para>
+
+                  <para>Nachdem Sie die Art der Quelle für Kursangebote ausgewählt haben, wählen Sie eine Datenquelle aus
+                    dem Pulldown-Menü. Die zur Verfügung stehenden Datenquellen sind abhängig von
+                    der installierten Version von &app-fq; und werden Ihnen bei Verwendung des
+                    Befehls <command>gnc-fq-check</command> angezeigt. Details zu den derzeit
+                    unterstützten Datenquellen finden Sie unter <xref linkend="fq-sources"></xref>.
+                    <note>
+<!-- Is this note still valid? -->
+                      <para>Die Kursdatenquelle <quote>Yahoo</quote> liefert dabei auch die Kurse für die übrigen genannten
+                        Unternehmen (z.B. Fidelity Investments, T.Rowe Price und The Vanguard Group)
+                        und gibt den Kurs normalerweise zeitverzögert zur Original-Datenquelle an.
```

Comment:
Die Problematik "Zeitverzögerung" hier rausziehen und in einem eigenen Absatz behandeln. Für Yahoo gibt es eine Tabelle gnc-tbl-fq-yahoo-exchange-codes, siehe auch NAV.